src/App.tsx

```diff
@@ -1,61 +1,103 @@
-import React from 'react';
+import { useState, FC } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { getReorderedGoods } from './functions';
+import { SortType } from './types';
+import goodsFromServer from './goodsFromServer';
+
+export const App: FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
+
+  const sortAlphabetically = () => {
+    if (sortType !== SortType.ALPHABET) {
+      setSortType(SortType.ALPHABET);
+    }
+  };
+
+  const sortByLength = () => {
+    if (sortType !== SortType.LENGTH) {
+      setSortType(SortType.LENGTH);
+    }
+  };
```

Comment:
Cool that, you have thought about those if statements (not necessary in this case but still cool)

You can leave it that way, but you could also go for a even better solution (hoping that it won't make the tests mad 😆)! Get rid off those if statements and disable buttons, basing on state:

`disabled={sortType === SortType.ALPHABET}`